src/AUNewsletterEpic/index.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactElement } from 'react';
+import { BrazeMessageComponent } from '../BrazeMessageComponent';
+import { StorybookWrapper } from '../utils/StorybookWrapper';
+import { withKnobs, text } from '@storybook/addon-knobs';
+
+export default {
+    component: 'AUNewsletterEpic',
+    title: 'WorkInProgress/EndOfArticle/AUNewsletterEpic',
+    decorators: [withKnobs({ escapeHTML: false })],
```

Comment:
I'm not sure we need the `escapeHTML: false` here. I think this is needed when you expect to pass HTML through to the component and render it as-is, which I don't think we do here.